spec/features/fill_in_whole_form_spec.rb

```diff
@@ -0,0 +1,180 @@
+
+    # Check Your Representative's Details Table
+    expect(confirmation_of_supplied_details_page).to have_confirmation_of_your_representatives_details_answers
+
+    your_rep_details_table = confirmation_of_supplied_details_page.confirmation_of_your_representatives_details_answers
+    expect(your_rep_details_table.type_of_representative_row.type_of_representative_answer).to have_text "Private Individual"
+    expect(your_rep_details_table.representative_org_name_row.representative_org_name_answer).to have_text "repco ltd"
+    expect(your_rep_details_table.representative_name_row.representative_name_answer).to have_text "Jane Doe"
+    expect(your_rep_details_table.representative_building_row.representative_building_answer).to have_text "Rep Building"
+    expect(your_rep_details_table.representative_street_row.representative_street_answer).to have_text "Rep Street"
+    expect(your_rep_details_table.representative_town_row.representative_town_answer).to have_text "Rep Town"
+    expect(your_rep_details_table.representative_county_row.representative_county_answer).to have_text "Rep County"
+    expect(your_rep_details_table.representative_postcode_row.representative_postcode_answer).to have_text "WC2 2BB"
+    expect(your_rep_details_table.representative_phone_row.representative_phone_answer).to have_text "0207 987 6543"
+    expect(your_rep_details_table.representative_mobile_row.representative_mobile_answer).to have_text "07987654321"
+    expect(your_rep_details_table.representative_dx_number_row.representative_dx_number_answer).to have_text "dx address"
+    expect(your_rep_details_table.representative_reference_row.representative_reference_answer).to have_text "Rep Ref"
+    expect(your_rep_details_table.representative_contact_preference_row.representative_contact_preference_answer).to have_text "Fax"
+    expect(your_rep_details_table.email_row.email_answer).to have_text nil
+    expect(your_rep_details_table.fax_row.fax_answer).to have_text "0207 345 6789"
+    expect(your_rep_details_table.representative_disability_row.representative_disability_answer).to have_text true
+    expect(your_rep_details_table.disability_information_row.disability_information_answer).to have_text "Lorem ipsum disability"
+
+    # Check Employer Contract Claim Table
+    expect(confirmation_of_supplied_details_page).to have_confirmation_of_employer_contract_claim_answers
+
+    employer_contract_claim_table = confirmation_of_supplied_details_page.confirmation_of_employer_contract_claim_answers
+    expect(employer_contract_claim_table.make_employer_contract_claim_row.make_employer_contract_claim_answer).to have_text true
+    expect(employer_contract_claim_table.claim_information_row.claim_information_answer).to have_text "lorem ipsum info"
+    expect(employer_contract_claim_table.upload_additional_information_row.upload_additional_information_answer).to have_text "sample.rtf"
+
+  end
+end
```

Comment:
Should rspec tests be this long? Does Rubocop pass on this?
If this is an end to end test, we can probably write a Cucumber test for it.

Reply:
It's the longest test I've written, not sure if it should/shouldn't be this size.

Rubocop passes on everything although it's been disabled on methods on other files (don't think it's been disabled in this file).

Reply:
@AlexaVB  - We dont want a mixture of cucumber and rspec for running end to end tests.  We need end to end tests as part of the rspec suite for BDD in general. Cucumber tests are more for QA to use like how we did it with the calculator.